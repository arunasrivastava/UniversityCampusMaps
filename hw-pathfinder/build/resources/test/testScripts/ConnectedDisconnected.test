# test dijkstra on 1 edge 3 nodes

CreateGraph g
AddNode g c1
AddNode g c2
AddNode g c3
AddEdge g c1 c2 2
AddEdge g c2 c1 2

FindPath g c1 c2

FindPath g c3 c1