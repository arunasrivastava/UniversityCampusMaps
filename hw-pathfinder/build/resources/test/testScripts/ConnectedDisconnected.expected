# test dijkstra on 1 edge 3 nodes

created graph g
added node c1 to g
added node c2 to g
added node c3 to g
added edge 2.000 from c1 to c2 in g
added edge 2.000 from c2 to c1 in g

path from c1 to c2:
c1 to c2 with weight 2.000
total cost: 2.000

path from c3 to c1:
no path found